desktop/core/src/desktop/js/reactComponents/createRootElements.js

```diff
@@ -31,7 +31,7 @@ async function render(name, props, root) {
 }
 
 export async function createReactComponents(selector) {
-  // Find all DOM containers
+  // Find all DOM containers .
```

Comment:
Nit: not required for the PR?

Reply:
No not required. It was purely for testing purposes